src/components/parameters/component-parameters-implementation.adb

```diff
@@ -290,6 +290,25 @@ package body Component.Parameters.Implementation is
       return Param_Update.Status;
    end Stage_Parameter;
 
+   -- Helper function to send a validation for a single components parameters:
+   function Validate_Parameters (Self : in out Instance; Component_Id : in Connector_Types.Connector_Index_Type) return Parameter_Enums.Parameter_Update_Status.E is
+      use Parameter_Enums.Parameter_Update_Status;
+      use Parameter_Enums.Parameter_Operation_Type;
+      -- Create a parameter update record:
+      Param_Validate : Parameter_Update.T := (
+         Operation => Validate,
+         Status => Success,
+         Param => (Header => (Id => 0, Buffer_Length => 0), Buffer => [others => 0]
+      ));
+      -- Component index:
+      Idx : constant Parameter_Update_T_Provide_Index := Parameter_Update_T_Provide_Index (Component_Id);
+   begin
+      -- Send the parameter fetch request to the appropriate component:
+      Self.Parameter_Update_T_Provide (Idx, Param_Validate);
```

Comment:
Need event if fail

Reply:
Added in [1e5756b](https://github.com/lasp/adamant/pull/65/commits/1e5756ba3fca1eb6e6c58154922a7be37504f8c3).